drivers/clock_control/clock_stm32_ll_h7.c

```diff
@@ -874,19 +1063,24 @@ int stm32_clock_control_init(const struct device *dev)
 	/* AHB/AXI/HCLK clock is SYSCLK / HPRE */
 	/* If freq increases, set flash latency before any clock setting */
 	if (new_hclk_freq > old_hclk_freq) {
-		LL_SetFlashLatency(new_hclk_freq);
+		LL_FLASH_SetLatency(LL_FLASH_LATENCY_6);
 	}
 
 	/* Preset the prescalers prior to choosing SYSCLK */
 	/* Prevents APB clock to go over limits */
 	/* Set buses (Sys,AHB, APB1, APB2 & APB4) prescalers */
 	LL_RCC_SetSysPrescaler(sysclk_prescaler(STM32_D1CPRE));
 	LL_RCC_SetAHBPrescaler(ahb_prescaler(STM32_HPRE));
+#if defined(CONFIG_SOC_SERIES_STM32H7RSX)
+	LL_RCC_SetAPB1Prescaler(apb1_prescaler(STM32_D1PPRE));
+	LL_RCC_SetAPB2Prescaler(apb2_prescaler(STM32_D2PPRE));
+	LL_RCC_SetAPB4Prescaler(apb4_prescaler(STM32_D4PPRE));
+	LL_RCC_SetAPB5Prescaler(apb5_prescaler(STM32_D5PPRE));
+#else
 	LL_RCC_SetAPB1Prescaler(apb1_prescaler(STM32_D2PPRE1));
 	LL_RCC_SetAPB2Prescaler(apb2_prescaler(STM32_D2PPRE2));
-	LL_RCC_SetAPB3Prescaler(apb3_prescaler(STM32_D1PPRE));
 	LL_RCC_SetAPB4Prescaler(apb4_prescaler(STM32_D3PPRE));
-
+#endif
```

Comment:
Changes impacting H7 series should be made at least in a dedicated commit, ideally in a dedicated PR

Reply:
remove change related to non-stm32h7rs